.cicd/build.sh

```diff
@@ -3,6 +3,11 @@ set -eo pipefail
 . ./.cicd/helpers/general.sh
 mkdir -p $BUILD_DIR
 CMAKE_EXTRAS="-DCMAKE_BUILD_TYPE='Release' -DENABLE_MULTIVERSION_PROTOCOL_TEST=true -DENABLE_UNITY_BUILD=ON"
+if [[ "$BUILDKITE" == 'true' ]]; then
+    SECRETS_DIR='/System/Volumes/Data/Network/NAS/MAC_FLEET/ANKA/secrets'
+    AMQP_CONNECTION_STRING=${AMQP_CONNECTION_STRING:-"$(cat $SECRETS_DIR/amqp-connection-string.txt)"}
+    CMAKE_EXTRAS="$CMAKE_EXTRAS -DAMQP_CONN_STR='"$AMQP_CONNECTION_STRING"'"
```

Comment:
Is this a different queue for each test run, or is it safe to have multiple tests connecting to the same queue?

Reply:
For this particular test it is safe for all tests to share the same queue. If we have more complicated setups in the future a diff scheme will likely be needed.

Reply:
Working on running these tests differently. One rabbitmq per test.